src/openforms/config/api/urls.py

```diff
@@ -10,4 +10,9 @@
         PrivacyPolicyInfoView.as_view(),
         name="privacy-policy-info",
     ),
+    path(
+        "declarations_info_list",
+        DeclarationsInfoListView.as_view(),
+        name="declarations-info-list",
+    ),
```

Comment:
convention is to use dashes rather than underscores in the URLs, but I see we have some more endpoints that are not adhering to this. I'll write it down for the v3 breaking changes.